third_party/3/werkzeug/wrappers.pyi

```diff
@@ -133,7 +133,7 @@ class ETagResponseMixin:
     def add_etag(self, overwrite=False, weak=False): ...
     def set_etag(self, etag, weak=False): ...
     def get_etag(self): ...
-    def freeze(self, no_etag=False): ...
+    def freeze(self, *, no_etag=False): ...
```

Comment:
I'm  unclear why this is needed for compatibility with a base class that defines freeze() without that extra parameter. Surely as long as there's a default it should be understood to be compatible with the base class?